guides/doc-Content_Management_Guide/master.adoc

```diff
@@ -42,6 +42,10 @@ include::topics/Using_ISS.adoc[]
 
 include::topics/Managing_Activation_Keys.adoc[]
 
+ifdef::satellite[]
+include::common/assembly_converting-a-host-with-convert2rhel.adoc[leveloffset=+1]
```

Comment:
Is this only useful for Satellite? I think it's okay to show this in every flavour.

Reply:
Removed if statement.